### ISSUE TITLE
%logger is reported as null

### ISSUE THREAD

**Issue by bsubburathinam** [rkcpi/logback-gelf-appender#16]
I have the following appender configured

```
<appender name="GELF" class="de.appelgriepsch.logback.GelfAppender">
    <server>my_server_dns</server>
    <port>12201</port>
    <protocol>UDP</protocol>
    <includeSource>true</includeSource>
    <includeMDC>true</includeMDC>
    <includeLevelName>true</includeLevelName>
    <queueSize>512</queueSize>
    <layout class="ch.qos.logback.classic.PatternLayout">
       <pattern>%date [%level] from %logger - %message%n%xException</pattern>
    </layout>
</appender>
```

Everything works fine except the message is reported as follows in Graylog:

`...[DEBUG] from null - Received blah blah blah...`

I am wondering if I am doing something wrong with the %logger as it is reported as "null".

**Comment by rkcpi** [comment]
If I understand correctly this should be fixed by #17. Can you confirm this?

**Comment by bsubburathinam** [comment]
This is fixed now. 👍 

`2018-02-21 11:42:57,035 [DEBUG] from x.y.zController - blah blah`
